wave_vision/include/wave/vision/common.hpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file
+ * Common computer vision utilities
+ * @ingroup vision
+ */
+#ifndef WAVE_VISION_COMMON_HPP
+#define WAVE_VISION_COMMON_HPP
+
+#include <opencv2/calib3d/calib3d.hpp>
+#include <opencv2/features2d/features2d.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/opencv.hpp>
+
+#include "wave/utils/utils.hpp"
+
+namespace wave {
+
+/** Calculate theoretical focal length based on field of view (fov) in radians
+ * and image width in pixels */
+double focal_length(double fov, double image_width);
+
+/** Calculate theoretical focal length based on field of view (fov) in
```

Comment:
Doxygen uses the first sentence as a "brief" description. So generally keep the first sentence to a few words, and describe the parameters with `@param` tags (or just on the next line).

Also, inconveniently, we need the `@addtogroup vision` tags inside the `namespace wave`.

I have made these changes.